examples/regionizers/slippy_map_regionizer.ipynb

```diff
@@ -0,0 +1,129 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Slippy Map Regionizer Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "from shapely import geometry\n",
+    "\n",
+    "from srai.regionizers import SlippyMapRegionizer\n",
+    "from srai.plotting.folium_wrapper import plot_regions\n",
+    "from srai.constants import WGS84_CRS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define starting polygons"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf = gpd.GeoDataFrame(\n",
+    "    geometry=[\n",
+    "        geometry.Polygon(\n",
+    "            shell=[\n",
+    "                (-1, 0),\n",
+    "                (0, 0.5),\n",
+    "                (1, 0),\n",
+    "                (1, 1),\n",
+    "                (0, 1),\n",
+    "            ],\n",
+    "            holes=[\n",
+    "                [\n",
+    "                    (0.8, 0.9),\n",
+    "                    (0.9, 0.55),\n",
+    "                    (0.8, 0.3),\n",
+    "                    (0.5, 0.4),\n",
+    "                ]\n",
+    "            ],\n",
+    "        ),\n",
+    "        geometry.Polygon(shell=[(-0.25, 0), (0.25, 0), (0, 0.2)]),\n",
+    "    ],\n",
+    "    crs=WGS84_CRS,\n",
+    ")\n",
+    "gdf.explore()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define tile resolution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "resolution = 10"
```

Comment:
I would change the name to zoom and increase it to 12 to better see the tesselation on the polygons
```suggestion
    "zoom = 12"
```